src/default.css

```diff
@@ -119,6 +119,7 @@ button {
 }
 
 input[type="submit"],
+input[type="button"],
```

Comment:
input[type=button] shouldn't be necesarry, pretty sure they do roughly the same (correct me if I'm wrong)

Reply:
en button af typen <input> er det samme som <button>, den kan bare ikke tage billeder som value lige som en button kan

Reply:
nvm min info er alt for gammel